contributing_to_docs/doc_guidelines.adoc

```diff
@@ -128,62 +128,41 @@ ID. For example:
 
-[IMPORTANT]
+[NOTE]
 ====
-Use a verb in the task title (for example, Creating).
+For `Prerequisites`, `Next steps`, and `Additional resources` headings, use `==` formatting in assembly files. Do not use `.Prerequisites`, `.Next steps`, or `.Additional information`, as this syntax should only be used within module files to suppress TOC formatting.
 ====
 
-Task modules take the following form:
-
-----
-<module_metadata>
-
-After the module file metadata, include a paragraph explaining why the user must
-perform this task. This should be 1-2 sentences maximum.
-
-If applicable, include any gotchas (things that could trip up the user or cause the task to fail).
-
-== Prerequisites
-
-* A bulleted list of prerequisites that MUST be performed before the user can complete this task.
-Skip if there isn't any related information.
-
-.Procedure
-
-. Step 1 - One command per step.
-
-. Step 2 - One command per step.
+== Writing concepts
+A _concept_ contains information to support the tasks that users want to do and
+must not include task information like commands or numbered steps. In most
+cases, create your concepts as individual modules and include them in
+appropriate assemblies.
 
-. Step N
+Avoid using gerunds in concept titles. "About <concept>"
+is a common concept module title.
 
-== Next steps
+For more information about creating concept modules, see the
+link:https://redhat-documentation.github.io/modular-docs/#creating-concept-modules[Red Hat modular docs reference guide] and the link:https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_CONCEPT_concept-explanation.adoc[concept template].
 
-You can explain any other tasks that MUST be completed after this task. You can
-skip this if there are none. Do not include xrefs. If the next steps are closely
-related to the task, you might be able to include their modules in the assembly.
+== Writing procedures
+A _procedure_ contains the steps that users follow to complete a process or task. Procedures contain ordered steps and explicit commands. In most cases, create your procedures as individual modules and include them in appropriate assemblies.
 
-.Related information
+Use a gerund in the procedure title, such as "Creating".
 
-* A bulleted list of links to related information about this task. Skip if there isn't any related information.
-----
+For more information about writing procedures, see the
+link:https://redhat-documentation.github.io/modular-docs/#creating-procedure-modules/[Red Hat modular docs reference guide] and the link:https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc[procedure template].
```

Comment:
The link doesn't work with the ending slash

Reply:
resolved